src/ota/ota_pal_test.c

```diff
@@ -81,10 +81,11 @@ TEST_TEAR_DOWN( Full_OTA_PAL )
 {
     OtaPalStatus_t xOtaStatus;
 
-    /* Abort the OTA file after every test. This closes the OTA file. */
+    /* Abort the OTA file after every test. This closes the OTA file.
+     * otaPal_Abort should only return OtaPalSuccess or OtaPalAbortFailed. */
     xOtaStatus = otaPal_Abort( &xOtaFile );
-
-    TEST_ASSERT_EQUAL_MESSAGE( OtaPalSuccess, xOtaStatus, "Failed to abort xOtaFile" );
+    TEST_ASSERT( OTA_PAL_MAIN_ERR( xOtaStatus ) == OtaPalSuccess ||
```

Comment:
Perhaps this assertion should be removed entirely? Does the OTA library require a particular set of return values?

Reply:
There are some other return values in [OtaPalMainStatus_t](https://www.freertos.org/Documentation/api-ref/ota-for-aws-iot-embedded-sdk/docs/doxygen/output/html/group__ota__enum__types.html#ga8e503bbbffc277f6fd5b8645767a685b). 
But I think we still can check the return value.